### ISSUE TITLE
Connecting to Server ...

### ISSUE THREAD

**Issue by PST-AG** [thecodingmachine/workadventure#4171]
Hi everyone,

I have deployed workadventure and jisti meet. Everything is working properly but the chat feature is not working, it always showing this error. When 2 user closing each other the "Proximity Chat" works fine. Would you please assist me?

![chat-issue](https://github.com/user-attachments/assets/95c2e7fc-c829-4882-adea-35114b8ccb52)

**Comment by moufmouf** [comment]
Hey @PST-AG ,

I guess you deployed the "master" branch which is not still tagged as a stable version.

What you see here is an attempt to connect to a Matrix server for messaging. But connecting to a Matrix server requires a bit more setup than what is currently written in the docs. It is work in progress.

We should also completely hide this "Connecting to server" block if no Matrix server is configured in your environment. Ping @ydiarra 

So basically, please be patient as we finish coding the Matrix integration :)
Or use the latest stable version tagged which is using the old XMPP server integration and that does not have this issue :D

**Comment by PST-AG** [comment]
Hi @moufmouf @ydiarra 

Could you kindly provide a stable GitHub branch with Synapse integration suitable for use in a production environment?

**Comment by PST-AG** [comment]
@moufmouf 
I am using an external keycloak server. Created two client id for workadventure(apache) and synapse. Now getting this error.
![image](https://github.com/user-attachments/assets/5ebc191e-6001-4b58-a4ac-e759abcd4700)
